src/TestData/Provider/RandomCurrency.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Give\TestData\Provider;
+
+class RandomCurrency extends RandomProvider {
+
+	/** @var array [currencyCode, ... ] */
+	protected $currencies = [
+		'EUR',
```

Comment:
@kakshak  GiveWP does not accept a donation in multiple currencies without currency switcher. We do not recommend admin to switch base currency multiple times on the website.
Why we are generating donations in multiple currencies?

Reply:
@ravinderk that is a good point. Since the Currency Switcher tracks the switched currency as meta it would make more sense to generate donations using a single currency, then have the option to generate a "switched currency" specifically.

Thank you for pointing this out.

Reply:
@ravinderk I have updated the related issue's acceptance criteria to address how switched currencies are supported.